apidocs/generate.py

```diff
@@ -0,0 +1,34 @@
+import os
+import re
+import pdoc
+
+from bytewax.execution import run_main, cluster_main
+
+context = pdoc.Context()
+
+bytewax = pdoc.Module("bytewax", context=context)
+pdoc.link_inheritance(context)
+
+# Use our templates
+pdoc.tpl_lookup.directories.insert(0, "templates")
+
+# HACK: These two functions aren't picked up by our module structure
+# as a part of bytewax.execution, so we inject them here directly.
+execution = bytewax.doc["execution"]
+execution.doc["run_main"] = pdoc.Function("run_main", execution, run_main)
+execution.doc["cluster_main"] = pdoc.Function("cluster_main", execution, cluster_main)
```

Comment:
Are there any other functions that this happens for? Can you describe the general rule for when we need to hack this in? It's any function that is defined via PyO3 that we re-import into a different module?